### ISSUE TITLE
Update staking docs examples to show how transaction fees are deducted

### ISSUE THREAD

**Issue by tjanez** [oasislabs/docs#215]
At the moment, examples in [_Operator Docs > Stake Management > Example transactions_](https://docs.oasis.dev/operators/stake-management.html#example-transactions) use a network where the same entity/account is used for the example transactions' source account and the validator node.

This results in the collected transactions fees being added to the same account, which is confusing to the reader as no transactions fees seem to be collected/deducted.

Update examples to use two distinct accounts for the validator node and example transactions' source account and show how transaction fees are deducted.

Additionally, the staking docs examples should use validator node configuration that doesn't have the `consensus.tendermint.min_gas_price` flag set zero so that it will not accept such low transaction fees (e.g. 1 base unit).

**Comment by pro-wh** [comment]
they shouldn't be reducing the offered gas price down to zero though

**Comment by tjanez** [comment]
> they shouldn't be reducing the offered gas price down to zero though

You are correct, thanks for pointing this out!

The example network where I ran these transactions used the same entity/account for the example transactions' source account and the validator node.

Hence, the collected transactions fees got added to the same account, resulting in seemingly no fees collected/deducted.

I've updated the issue's description.
